index.html

```diff
@@ -421,6 +421,11 @@ <h2>Terminology</h2>
           (linked data) object.
         </dd>
 
+        <dt>target</dt>
```

Comment:
Why was this renamed from "invocationTarget" to just "target"? I think it's more likely to conflict with other vocabularies making plain JSON implementations more difficult.

Reply:
There is no invocationTarget or any such target property anymore: since the target is the root capability, we now use parentCapability to point to it.  So this just refers to the concept of a target, not a target property.

Maybe there's a way to make that clearer?  What do you think?